async_downloads/views.py

```diff
@@ -25,6 +22,9 @@ def ajax_update(request):
             dl["url"] = default_storage.url(dl["filepath"])
         else:
             in_progress = True
+        if WS_MODE:
+            dl["timestamp"] = str(dl["timestamp"])
+            dl["downloadKey"] = download_key
```

Comment:
We have `downloadKey` and `css_class` - can we stick to one naming convention, either camelCase or snake_case? (I don't mind which here, though I probably favour snake_case 🙂.)

Reply:
05bab57